### PR TITLE
Add stream API Test coverage

### PR DESCRIPTION
**Pull request by timbob1000**

### CODE REVIEW

File: onyx-database-tests/src/streams/BasicQueryMapStream.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import com.onyx.stream.QueryMapStream;
+import entities.identifiers.ImmutableSequenceIdentifierEntity;
+import entities.identifiers.ImmutableSequenceIdentifierEntityForDelete;
+
+import java.util.Map;
+
+/**
+ * Created by tosborn1 on 7/14/16.
+ */
+public class BasicQueryMapStream implements QueryMapStream{
```
Comment (cosbor11) [inline onyx-database-tests/src/streams/BasicQueryMapStream.java]: QureyMapStream should be named IQueryMapStream

Comment (timbob1000) [inline onyx-database-tests/src/streams/BasicQueryMapStream.java]: I thought we agreed on this convention a long while back that the interfaces should not be prefixed with I.  Such as PersistenceManager is not IPersistenceManager and the default implementation should be DefaultPersistenceManager.  

The same applies here.

Comment (cosbor11) [inline onyx-database-tests/src/streams/BasicQueryMapStream.java]: What about IManagedEntity?

Comment (timbob1000) [inline onyx-database-tests/src/streams/BasicQueryMapStream.java]: That breaks the convention and should probably be refactored at some point

**Pull request merged**